src/screens/Routing/History.res

```diff
@@ -1,13 +1,13 @@
 open HistoryEntity
 module HistoryTable = {
   @react.component
-  let make = (~records, ~activeRoutingIds: array<string>) => {
+  let make = (~records, ~activeRoutingIds: array<string>, ~customTitle) => {
```

Comment:
can you give the default `title=" "`, because empty title throws error in this component